web/consumers.py

```
@@ -0,0 +1,207 @@
+                        'type': 'participant_joined',
+                        'user': {
+                            'username': self.user.username,
+                            'full_name': self.user.get_full_name() or self.user.username,
+                        },
+                        'seat_id': participant.seat_id
+                    }
+                )
+
+                # Send current participants list to everyone
+                await self.send_participants_list()
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)